README.md

```diff
@@ -53,7 +58,10 @@ The preview website url will be `https://{{repository.owner}}-{{repository.name}
 ```yaml
 name: 🔂 Surge PR Preview
 
-on: [push, pull_request_target]
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, closed]
```

Comment:
Don't change this, other types should only be enabled when using `teardown`.

Reply:
You can instruct that in `teardown` option.

Reply:
yup tht makes sense, ill also add a note expressing the same

Reply:
updated, can review again